backend/rds.ts

```diff
@@ -154,7 +154,7 @@ async function executeStatementInDatabase(
       name,
       value,
     }));
-  console.log("Executing", database, sql, params);
+  console.log("Executing", database, sql, JSON.stringify(params, null, ''));
```

Comment:
These were creating too much noise in the logs pretty-printed. There's probably some other way to do this but it works for now.